### DISCUSSION TITLE
HW 4 Discussion

### DISCUSSION THREAD

**Discussion by adamscottc** [williamgilpin/cphy#19] [homework]
Hi! I started working on HW 4 today, and I was working on deriving the non-dimensionalized Navier-Stokes equation, but I had several questions that were stumping me. Did we get rid of the external force term F and how does the parenthetical tensor term on the rhs simplify to a Laplacian?

**Comment by adamscottc** [comment]
Also, what kind of norm is the matrix norm that is given in the scalar loss function?

**Comment by williamgilpin** [comment]
@adamscottc Hi, I just updated the notebook to include the two vector calc identities used in the reduction to incompressible Navier-Stokes. There was also a typo in my definition of the incompressibility condition, but I am also assuming a zero body force. For the loss function, I put the MSE into explicit vector form, which makes the derivation a little easier to see
